pkg/machinery/config/types/v1alpha1/v1alpha1_types.go

```diff
@@ -1261,6 +1261,9 @@ type APIServerConfig struct {
 
 // ControllerManagerConfig represents the kube controller manager configuration options.
 type ControllerManagerConfig struct {
+	//   description: |
+	//     Disable clusterControllerManager on the node.
+	Disabled bool `yaml:"disabled,omitempty"`
```

Comment:
I feel like probably this should part of the machine configuration, not cluster one, as we certainly don't want disable things on cluster level

Reply:
Proposal config:

```yaml
machine:
  kubelet:
    staticPod:
      kube-controller-manager:
        disable: true
      kube-scheduler: {}
```